blocks/core/_lib/ff-tabs/_ff-tabs-core.js

```diff
@@ -91,12 +90,38 @@ function getTabHandler($root, options) {
 
     function init() {
         $root.on('click', linkSel, handleClick);
-        setInitialTab();
+        var index = getHashIndex() || 0;
+        setActiveTab(index);
         /*jshint validthis:true */
         return main;
     }
 
-    function setState(target) {
+    function getHashIndex() {
+        var hash = getHash();
+        if (hash) return getIndexOfTrigger(hash);
+        return null;
+    }
+
+    function getHash() {
+        var hash = window.location.hash.substring(1);
+        if (hash) return hash;
+        return null;
+    }
+
+    function getIndexOfTrigger(hash) {
+        var $triggers = $root.find(linkSel);
+        var index = $($triggers).filter(function(i) {
```

Comment:
$triggers should be a jQuery object already, so you shouldn't have to wrap it again to filter? I could be wrong, but worth checking to avoid an additional instantiation.